2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_setup.py:_flush():71] setting env: {}
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_init.py:_log_setup():371] Logging user logs to /home/beegass/Documents/Coding/Readable-VAEs/vae-pytorch/wandb/run-20211207_203809-dwafmq58/logs/debug.log
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_init.py:_log_setup():372] Logging internal logs to /home/beegass/Documents/Coding/Readable-VAEs/vae-pytorch/wandb/run-20211207_203809-dwafmq58/logs/debug-internal.log
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_init.py:init():404] calling init triggers
2021-12-07 20:38:09,242 INFO    MainThread:424592 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-07 20:38:09,243 INFO    MainThread:424592 [wandb_init.py:init():449] starting backend
2021-12-07 20:38:09,243 INFO    MainThread:424592 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-07 20:38:09,249 INFO    MainThread:424592 [backend.py:ensure_launched():199] starting backend process...
2021-12-07 20:38:09,254 INFO    MainThread:424592 [backend.py:ensure_launched():204] started backend process with pid: 424641
2021-12-07 20:38:09,255 INFO    MainThread:424592 [wandb_init.py:init():458] backend started and connected
2021-12-07 20:38:09,256 INFO    MainThread:424592 [wandb_init.py:init():519] updated telemetry
2021-12-07 20:38:09,268 INFO    MainThread:424592 [wandb_init.py:init():549] communicating current version
2021-12-07 20:38:10,053 INFO    MainThread:424592 [wandb_init.py:init():554] got version response 
2021-12-07 20:38:10,054 INFO    MainThread:424592 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-07 20:38:10,539 INFO    MainThread:424592 [wandb_init.py:init():592] starting run threads in backend
2021-12-07 20:38:10,567 INFO    MainThread:424592 [wandb_run.py:_console_start():1816] atexit reg
2021-12-07 20:38:10,568 INFO    MainThread:424592 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-07 20:38:10,568 INFO    MainThread:424592 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-07 20:38:10,569 INFO    MainThread:424592 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-07 20:38:10,569 INFO    MainThread:424592 [wandb_init.py:init():619] run started, returning control to user process
2021-12-07 21:11:12,768 INFO    MainThread:424592 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2021-12-07 21:11:12,768 INFO    MainThread:424592 [wandb_run.py:_restore():1758] restore
2021-12-07 21:11:14,965 INFO    MainThread:424592 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 756
}

